src/omnibar/omnibar.ts

```diff
@@ -71,6 +75,10 @@ let omnibarConfig: BBOmnibarConfig;
 let currentLegacyKeepAliveUrl: string;
 let promiseResolve: () => void;
 let pushNotificationsConnected: boolean;
+let unreadNotificationCount: number;
+let serviceName: string;
+let currentTitleParts: string[] = [];
```

Comment:
This doesn't need to be initialized if you follow the suggestion in the comment on `titleSet` line below.